cabal-install/Distribution/Client/ProjectConfig.hs

```diff
@@ -480,19 +474,22 @@ readProjectLocalExtraConfig verbosity distDirLayout =
 readProjectLocalFreezeConfig :: Verbosity -> DistDirLayout
                              -> Rebuild ProjectConfig
 readProjectLocalFreezeConfig verbosity distDirLayout =
-    readProjectExtensionFile verbosity distDirLayout "freeze"
+    readProjectFile verbosity distDirLayout "freeze"
                              "project freeze file"
 
 -- | Reads a named config file in the given project root dir, or returns empty.
 --
-readProjectExtensionFile :: Verbosity -> DistDirLayout -> String -> FilePath
-                         -> Rebuild ProjectConfig
-readProjectExtensionFile verbosity DistDirLayout{distProjectFile}
+readProjectFile :: Verbosity
+                -> DistDirLayout
+                -> String
+                -> String
```

Comment:
I think this used to be FilePath

Reply:
Which is clearly wrong, is it not? It is the `extensionDescription` after all, a string meant for displaying purposes.

Reply:
yeah, sorry